modules/nf-core/sentieon/bwamem/main.nf

```diff
@@ -3,7 +3,9 @@ process SENTIEON_BWAMEM {
     label 'process_high'
     label 'sentieon'
 
-    secret 'SENTIEON_LICENSE_BASE64'
+    // NOTE this is not required for the process to run, but it is required for the process to be run in GitHub actions or nf-core MegaTests
+    // The rest of the secrets aren't really "secrets" because they're not sensitive information for users
+    // secret SENTIEON_AUTH_DATA
```

Comment:
is this really required as all is commented out?

Reply:
Eh, let's leave it in so it's in the git history, and then we can cut it out. I have another PR to do some fancier clean up. This is just an MVP.